web/src/components/TimeDisplay.tsx

```diff
@@ -1,14 +1,16 @@
-import { useAtom } from 'jotai';
+import { useAtomValue } from 'jotai/react';
```

Comment:
Why do we import `useAtomValue` from `jotai` sometimes and `jotai/react` other times?

Reply:
Honestly I'm not sure, this is the import path that was suggested by VSCode. I can change it back and align them in another PR.